react/features/toolbox/types.ts

```diff
@@ -49,3 +48,8 @@ export enum NOTIFY_CLICK_MODE {
     ONLY_NOTIFY = 'ONLY_NOTIFY',
     PREVENT_AND_NOTIFY = 'PREVENT_AND_NOTIFY'
 }
+
+export type MAIN_TOOLBAR_BUTTONS_THRESHOLDS = Array<{
```

Comment:
since this is a type, please rename to it `IMainToolbarButtonThresholds`

Reply:
Fixed